packages/task/src/browser/task-service.ts

```diff
@@ -1008,11 +1018,14 @@ export class TaskService implements TaskConfigurationClient {
                 taskConfig: taskInfo ? taskInfo.config : undefined
             }
         );
-        widget.start(processId);
+        widget.start(terminalId);
     }
 
-    private getTerminalWidgetId(terminalId: number): string {
-        return `${TERMINAL_WIDGET_FACTORY_ID}-${terminalId}`;
+    private getTerminalWidgetId(terminalId: number): string | undefined {
```

Comment:
could you change the modifier `private` to `protected`

Reply:
yep, updated the code as suggested in the latest patch. Thank you for the feedback !

Reply:
thanks!